tests/integration/telemetry/stackdriver/vm/main_test.go

```diff
@@ -63,6 +64,13 @@ func TestMain(m *testing.M) {
 			cfg.Values["telemetry.v2.enabled"] = "true"
 			cfg.Values["telemetry.v2.stackdriver.enabled"] = "true"
 			cfg.Values["telemetry.v2.stackdriver.logging"] = "true"
+			cfg.ControlPlaneValues = `values:
```

Comment:
Is it that flattening this as line 63 - 67 does not work?

Reply:
I think above lines have no effect if configOverride is provided..

Reply:
it was that i was able and wanted to copy/paste from another place. updated to match the lines above.